sample_apps/islet_attest_seal/attest_seal_app.cc

```diff
@@ -0,0 +1,79 @@
+#include <islet.h>
+
+#include <iostream>
+#include <string>
+#include <cstring>
+
+using byte = unsigned char;
+
+static const char CLAIM_TITLE_USER_DATA[] = "User data";
+static const char CLAIM_TITLE_PLATFORM_PROFILE[] = "Profile";
+
+bool attestation_test() {
+    byte report[2048], claims[1024], value[1024];
+    memset(report, 0, sizeof(report));
+    memset(claims, 0, sizeof(report));
+    memset(value, 0, sizeof(value));
+    int report_len = 0, claims_len = 0, value_len = 0;
+
+    // -- Attest -- //
+    std::string user_data("User Custom data");
+    if (islet_attest((const byte*)user_data.c_str(), user_data.size(), report, &report_len))
+        return false;
+
+    // -- Verify -- //
+    if (islet_verify(report, report_len, claims, &claims_len))
+        return false;
+
+    islet_print_claims(claims, claims_len);
+
+    // -- Parse -- //
+    if (islet_parse(CLAIM_TITLE_USER_DATA, claims, claims_len, value, &value_len))
+        return false;
+
+    printf("Claim[User data]: %s\n", (char*) value);
+
+    memset(value, 0, sizeof(value));
+    if (islet_parse(CLAIM_TITLE_PLATFORM_PROFILE, claims, claims_len, value, &value_len))
+        return false;
+
+    printf("Claim[Platform  profile]: %s\n", (char*) value);
+
+    return true;
+}
+
+bool sealing_test() {
+    byte sealed[2048], unsealed[2048];
+    memset(sealed, 0, sizeof(sealed));
+    memset(unsealed, 0, sizeof(unsealed));
+    int sealed_len = 0, unsealed_len = 0;
+
+    // -- Seal -- //
+    std::string plaintext("Plaintext");
+    if (islet_seal((const byte*)plaintext.c_str(), plaintext.size(), sealed, &sealed_len))
+        return false;
+
+    // -- Unseal -- //
+    if (islet_unseal(sealed, sealed_len, unsealed, &unsealed_len))
+        return false;
+
+    printf("Success sealing round trip.\n");
+
+    return true;
+}
+
+int main() {
+    printf("[!] Starting attestation test...\n");
+    if (attestation_test()) 
+        printf("[!] Success to attestation test...\n");
+    else
+        printf("[!] Failed to attestation test...\n");
+
+    printf("[!] Starting sealing test...\n");
+    if (sealing_test()) 
+        printf("[!] Success to sealing test...\n");
+    else
+        printf("[!] Failed to sealing test...\n");
```

Comment:
Please rewrite this block as suggested previously, using `bool rv` and ternary for `printf()`.